command/helper/helper.go

```diff
@@ -345,9 +345,11 @@ func BootstrapDevCommand(baseCommand string, args []string) (*Config, error) {
 	flags.Usage = func() {}
 
 	var premine helperFlags.ArrayFlags
+	var gaslimit uint64
 
 	flags.StringVar(&cliConfig.LogLevel, "log-level", DefaultConfig().LogLevel, "")
 	flags.Var(&premine, "premine", "")
+	flags.Uint64Var(&gaslimit, "gas-limit", 5000, "")
```

Comment:
If you can, please move `5000` to constants at the beginning of the file. You can name it something like `DefaultGasLimit`

Reply:
Done